lang-dotnet/dotnet-runtime-deps-8.0/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=dotnet-runtime-deps-8.0
+PKGSEC=libs
+PKGDES="dotnet-runtime-deps-aoscos 8.0.0"
```

Comment:
What?

Reply:
You should not create a meta package just to offer dependencies. Please specify these dependencies in each package.